asu/package_changes.py

```diff
@@ -42,3 +52,12 @@ def _add_if_missing(package):
 
             elif build_request.profile == "buffalo_wzr-hp-g300nh-rb":
                 _add_if_missing("kmod-switch-rtl8366rb")
+
+    # TODO: if we ever fully implement 'packages_versions', this needs rework
+    for version, packages in language_packs.items():
+        if build_request.version >= version:  # Includes snapshots
+            for i, package in enumerate(build_request.packages):
+                for old, new in packages.items():
+                    if package.startswith(old):
+                        lang = package.rsplit("-", 1)[1]
```

Comment:
this does not work if the language pack contains addtional `-`, for example luci-i18n-opkg-`zh-cn` will be evaluated as `cn`, then result the following error:
```
Collected errors:
 * opkg_install_cmd: Cannot install package luci-i18n-package-manager-cn.
 *
```

Reply:
@efahl can you please have a look?

Reply:
Thinking a simple fix could be `lang = package.replace(old, "")`